### ISSUE TITLE
[BUG] pre-post Segmentor does not segment resection cavity

### ISSUE THREAD

**Issue by zsdqui** [BrainLesion/BraTS#124]
### Describe the bug
On multiple test case, the BraTS Pre and post-treatment Segmentor fails to assign label "4" Resection cavity

### Steps to Reproduce
Steps to reproduce the behavior (for :
1. Download BraTS 2025 training data
2. use python or basic CLI wrapper as below

```python
import argparse
from brats import AdultGliomaPreAndPostTreatmentSegmenter
from brats.constants import AdultGliomaPreAndPostTreatmentAlgorithms

def main():
    # Set up argument parser
    parser = argparse.ArgumentParser(description='Run BraTS Orchestrator segmentation on glioma MRI data.')
    parser.add_argument('--t1c', type=str, required=True, help='Path to T1-contrast (T1c) NIfTI file')
    parser.add_argument('--t1n', type=str, required=True, help='Path to T1-native (T1n) NIfTI file')
    parser.add_argument('--t2f', type=str, required=True, help='Path to T2-FLAIR (T2f) NIfTI file')
    parser.add_argument('--t2w', type=str, required=True, help='Path to T2-weighted (T2w) NIfTI file')
    parser.add_argument('--output', type=str, required=True, help='Path to output segmentation NIfTI file')
    parser.add_argument('--algorithm', type=str, default="BraTS25_1",
                        choices=[e.value for e in AdultGliomaPreAndPostTreatmentAlgorithms],
                        help='Segmentation algorithm to use (default: BraTS25_1)')
    parser.add_argument('--cuda', type=str, default='0', help='CUDA device(s) to use (default: 0)')

    # Parse arguments
    args = parser.parse_args()

    # Initialize segmenter with chosen algorithm and CUDA device
    segmenter = AdultGliomaPreAndPostTreatmentSegmenter(
        algorithm=AdultGliomaPreAndPostTreatmentAlgorithms[args.algorithm],
        cuda_devices=args.cuda
    )

    # Perform segmentation
    segmenter.infer_single(
        t1c=args.t1c,
        t1n=args.t1n,
        t2f=args.t2f,
        t2w=args.t2w,
        output_file=args.output
    )

if __name__ == "__main__":
    main()
```
3. Run commands:
```sh
python run_brats.py --t1n /data/BraTS-GLI-00005-100/BraTS-GLI-00005-100-t1n.nii.gz --t1c /data/BraTS-GLI-00005-100/BraTS-GLI-00005-100-t1c.nii.gz --t2w /data/BraTS-GLI-00005-100/BraTS-GLI-00005-100-t2w.nii.gz  --t2f /data/BraTS-GLI-00005-100/BraTS-GLI-00005-100-t2f.nii.gz --output ~/segment.nii.gz
```

### Expected vs. actual behavior
The script routinely segments compartments 1-3 (enhancing tumor, necrotic core, flair region) but never seems to segment label 4 (resection cavity)

### Screenshots
(BraTS training case BraTS-GLI-0005-100 example, T1 image, orchestrator prediction, ground truth)
<img width="1087" height="435" alt="Image" src="https://github.com/user-attachments/assets/913ffacc-5995-45e9-a5f1-adde73547f79" />

### operating system and version?
Ubuntu 22.04 on AWS

### NVIDIA drivers and GPUs
please paste the output of (or a more suitable base version for your system):
```sh
nvidia-smi
```

<img width="614" height="318" alt="Image" src="https://github.com/user-attachments/assets/898c7e8e-f2e8-4561-98f9-926b064e531a" />

### Python environment and version?
e.g. Conda environment with Python 3.10. Check your Python version with:
```sh
python --version
```
Python venv
Python 3.10.12

### Docker version
please specify your docker version, You can find it by running: 
```sh
docker --version
```
Docker version 28.5.1, build e180ab8

### Version of brats ?
please specify your version of brats (please make sure you run the latest version):
```sh
pip freeze | grep brats
```

brats==0.1.2

**Additional context**

**Comment by zsdqui** [comment]
I isolated the issue to the yaml file: [/brats/data/meta/adult_glioma_pre_and_post_treatment.yml](https://github.com/BrainLesion/BraTS/blob/main/brats/data/meta/adult_glioma_pre_and_post_treatment.yml)
```yaml
constants:
  input_name_schema: &input_name_schema "BraTS-GLI-{id:05d}-000"
```

The hard-coding of all files with the suffix -000 causes the pre/post docker instances to treat all inputs as 'pre-surgery' and omit the resection cavity = 4 label. I tested on my private fork that changing the suffix to -100 (indicating 'post-surgery) fixes the issue. This would be an easy fix, but perhaps the name of this tool should that just be "Adult glioma post-treatment" (Breaking the alignment with how BraTS designated algorithms - but it seems that for an orchestrator a pre-treatment and post-treatment tool would be a better interface). Alternatively a flag could be added that changes the naming convention based on pre- vs. post-treatment scan, but that seems unwieldy.
